src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -302,11 +225,16 @@ function BaseOptionsSelector(props) {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [allOptions, focusedIndex, props.canSelectMultipleOptions, props.onConfirmSelection, selectRow]);
 
+    const activeElementRole = useActiveElementRole();
     useKeyboardShortcut(CONST.KEYBOARD_SHORTCUTS.ENTER, selectFocusedOption, {
         shouldBubble: !allOptions[focusedIndex],
         captureOnInputs: true,
+        isActive: !activeElementRole || activeElementRole === CONST.ROLE.PRESENTATION,
```

Comment:
I think we need to consider `isFocused` value here, otherwise it seems to be active on navigating to another page. we have the issue [here](https://github.com/Expensify/App/issues/37294)

Reply:
thanks, I'll add that now